h/accounts/test/views_test.py

```diff
@@ -830,30 +829,28 @@ def test_activate_event_when_validation_succeeds(event,
 
 
 @pytest.mark.usefixtures('subscriptions_model')
-def test_profile_looks_up_by_logged_in_user(authn_policy, user_model):
-    """
-    When fetching the profile, look up email for the logged in user.
+def test_profile_looks_up_by_logged_in_user():
```

Comment:
It's testing that the new `.authenticated_user` property gets called, previously it was testing that `get_by_userid()` got called, it's equivalent no?